src/core/absorption_chiller.jl

```diff
@@ -28,61 +28,217 @@
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 # *********************************************************************************
 
+heat_transfer_mediums = ["steam", "hot_water"]
+chp_prime_movers = ["recip_engine", "micro_turbine", "combustion_turbine", "fuel_cell"]
```

Comment:
In the commit 298eac373f9feec60c770487aa4089131721aece this is addressed; I also made a const for hot_water_or_steam since it's used by CHP too.  When the CHP work is merged, I'll update that side as well (I'm trying to avoid merge conflicts for now since CHP development is happening in parallel)